uber/decorators.py

```diff
@@ -210,13 +210,13 @@ def cached(func):
 
 
 def cached_page(func):
```

Comment:
This _looks_ like I changed a lot, but I really just moved the `if hasattr(innermost, 'cached'):` conditional to _outside_ the wrapped function. If the innermost function is not cached, then we just return the original function, undecorated. Thus `@cached_page` just becomes a NO-OP for any function not marked as cached.